pkg/api/v1/types.go

```diff
@@ -2011,6 +2013,20 @@ type ComponentStatusList struct {
 	Items []ComponentStatus `json:"items" description:"list of component status objects"`
 }
 
+// DownwardAPIVolumeSource represents a volume containing downward API info
+type DownwardAPIVolumeSource struct {
+	// Items is a list of downward api volume file
+	Items []DownwardAPIVolumeFile `json:"items,omitempty" description:"list of downward api files"`
+}
+
+// DownwardAPIVolumeFile represents information to create the file containing the pod field
+type DownwardAPIVolumeFile struct {
+	// Required: Name is the name of the file
+	Name string `json:"name" description:"the relative path name of the file to be created. Must not be absolute or contain the '..' path. Must be utf-8 encoded."`
```

Comment:
@bgrant0607 that makes sense to me.  What about `Path` for the name of this field?